src/index.ts

```diff
@@ -83,6 +83,34 @@ class Trie<T> {
 
     return results;
   }
+
+  /**
+   * Returns an array of values that have keys matching the query exactly.
+   *
+   * You are encouraged to pass an options object with:
+   * a .limit to limit the number of results returned.
+   * a .unique property if you only want one result per-key.
+   *
+   * The limit is particularly important because the performance of the
+   * algorithm is determined primarily by the limit.
+   */
+  exactSearch(query: string, opts?: Partial<SearchOptions>): T[] {
```

Comment:
I considered refactoring part of this out with a helper that would also match things with `prefixSearch`. But I think it would have only been confusing.